instance/models/server.py

```diff
@@ -477,3 +482,17 @@ def _shutdown(self, os_server, poll_interval=10, max_wait=None):
             max_wait -= poll_interval
             os_server = self.os_server
         return os_server.status == 'SHUTOFF'
+
+    def _delete_ssh_key(self) -> None:
+        """
+        Delete SSH key from `~/.ssh/known_hosts`.
+
+        We can safely ignore the command's return code, because we just need to be sure that the key has been removed
+        for non-existing server - we don't care about non-existing keys.
+        """
+        self.logger.info('Deleting SSH key of "%s" host.', self.public_ip)
+        command = f'ssh-keygen -R {self.public_ip}'
+        try:
+            subprocess.Popen(command, shell=True)
```

Comment:
@Agrendalath, nit: This logs the output of the command to the Ocim logs/console output. I think we should suppress this.

Reply:
@lgp171188, done.

Reply:
@lgp171188, done.